### ISSUE TITLE
controller installation: docker related error logged

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#255]
* master_installer
42c4bc2

During controller installation, ssh controller02
inspect /var/log/xcat/xcat.log

```
Error: Package: docker-1.6.0-11.0.1.el7.centos.x86_64 (otherpkgs1)
           Requires: device-mapper-libs >= 7:1.02.90-1
           Installed: 7:device-mapper-libs-1.02.84-14.el7.x86_64 (@centos7.0-x86_64-0/$releasever)
               device-mapper-libs = 7:1.02.84-14.el7
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
yum invocation failed
```

**Comment by hansthen** [comment]
Problem was that the oldest cd was being used for copycd, instead of the newest.

**Comment by hansthen** [comment]
Is also merged into master_installer.

**Comment by themisa** [comment]
fixed in *master_installer 1030300. 
Need to test also in r3
